xarray/core/dataarray.py

```diff
@@ -6670,6 +6675,69 @@ def resample(
             **indexer_kwargs,
         )
 
+    def to_dask_dataframe(
+        self, dim_order: Sequence[Hashable] = None, *, name: Hashable = None
+    ) -> DaskDataFrame:
+        """Convert this array into a dask.dataframe.DataFrame.
+
+        Parameters
+        ----------
+
+        dim_order: Sequence of Hashable or None , optional
+        Hierarchical dimension order for the resulting dataframe.
+        Array content is transposed to this order and then written out as flat
+        vectors in contiguous order , so the last dimension in this list
+        will be contiguous in the resulting DataFrame. This has a major influence on
+        which operations are efficient on the resulting dask dataframe.
+
+        name: Hashable or None, optional
+        Name given to this array(required if unnamed).
+        It is a keyword-only argument. A keyword-only argument can only be passed to the
+        function using its name as a keyword argument , and not as a positional argument.
+
+        Returns
+        -------
+        dask.dataframe.DataFrame
+
+        Examples
+        --------
+
+        da=xr.DataArray(np.random.rand(4,3,2),
```

Comment:
This example is not correctly formatted, see other functions as reference.

Also, don't use random values in examples, simply use `np.ones(...)`